qemu/tests/block_hotplug_in_pause.py

```diff
@@ -178,16 +220,18 @@ def rw_disk_in_guest(session, plug_disks, iteration):
             if params.get("stop_vm_before_hotplug", "no") == "yes":
                 error_context.context("Stop VM before hotplug")
                 vm.pause()
+                is_vm_paused = True
```

Comment:
I wonder why add variable is_vm_paused , does vm.is_paused() work?

Reply:
The `vm.is_paused()` does work, and for lines 188 and 205 don't need to check whether VM is paused by sending qmp command "query-status". It's redundant.

Reply:
Since the `vm.is_paused()` -> `self.verify_status("paused")` -> `self.monitor.verify_status(status)` -> `o = dict(self.cmd(cmd="query-status", debug=False))`